### PR TITLE
release(v0.9.0-beta.1): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.9.0-beta.1 release.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Review by rsmitty — APPROVED** [review]

File: hack/release.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+[notes]
+
+    [notes.features]
+        title = "New Features"
+        description = """\
+* Control Plane as Staitc Pods
+* ECDSA Keys for Kubernetes PKI
+* Disk Encryption
+* Virtual Shared IP for Control Plane Endpoint
```
Comment (Unix4ever) [inline hack/release.toml:24]: Maybe should also add ipv6 and wireguard support here.

**Review by Unix4ever — APPROVED** [review]

File: hack/release.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+preface = """\
+"""
+
+[notes]
+
+    [notes.features]
+        title = "New Features"
+        description = """\
+* Control Plane as Staitc Pods
```
Comment (Unix4ever) [inline hack/release.toml:21]:
```suggestion
* Control Plane as Static Pods
```

File: hack/release.toml (hunk 1 above)
Comment (smira) [reply hack/release.toml:24]: I was simply following https://www.talos.dev/docs/v0.9/introduction/what-is-new/, we can update it for the next release if we have more docs, there's nothing to explain what DHCPv6 and Wireguard is

File: hack/release.toml (hunk 2 above)
Comment (smira) [reply hack/release.toml:21]: :facepalm: thanks!

File: hack/release.toml (hunk 1 above)
Comment (Unix4ever) [reply hack/release.toml:24]: ah, I see 👍

**Review by Unix4ever — APPROVED** [review]

**Review by andrewrynhard — APPROVED** [review]

**Pull request merged**